distributed/worker.py

```diff
@@ -1731,43 +1737,59 @@ async def get_data(
     # Local Execution #
     ###################
 
+    @functools.singledispatchmethod
+    def handle_event(self, ev: StateMachineEvent) -> RecsInstrs:
+        raise TypeError(ev)  # pragma: nocover
+
```

Comment:
Moved so that I didn't have to move the various refactored functions, thus minimising the diff.
This is temporary - this and all the registered methods will be moved to worker_state_machine.py.